core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalCheckpointThread.java

```diff
@@ -205,6 +216,7 @@ private void runInternal() {
             }
             break;
           default:
+            mReplayState = ReplayState.REPLAY_DONE;
```

Comment:
What if the current log file is done (file1), but the next one (file2) was not complete yet. Would the replay be considered done after file1, or after file2 is completed and read by this master?

Reply:
UfsJournalCheckpointThread keeps catching up the "complete" journal files and ignores "incomplete" files. so only file 1 will be read and replayed. After that, the replay will be considered done and the checkpoint thread may start writing checkpoint. 
We will not catch up all journals, but catch up all complete journals.

Reply:
Yeah, that makes sense, and it should be fine to only have 1 incomplete file to replay afterwards.